app/client/sagas/userSagas.js

```diff
@@ -87,6 +88,30 @@ function* watchEditUser() {
   yield takeEvery(EDIT_USER_REQUEST, editUser);
 }
 
+function* resetPin({userId}) {
```

Comment:
did i mention we should do some refactoring?

Reply:
added todo: https://trello.com/c/cGlzTuiu/137-refactor-web-reducers-sagas